ci/authn-k8s/test_oc_entrypoint.sh

```diff
@@ -22,8 +22,8 @@ function finish {
 
   sleep 5
 
-  oc adm policy remove-scc-from-user anyuid -z default
-  oc --ignore-not-found=true delete project $CONJUR_AUTHN_K8S_TEST_NAMESPACE
+  #oc adm policy remove-scc-from-user anyuid -z default
+  #oc --ignore-not-found=true delete project $CONJUR_AUTHN_K8S_TEST_NAMESPACE
```

Comment:
Are these really not needed anymore? (I'm most curious about not deleting the project/namespace)

If not, can we delete the lines rather than commenting them.

Reply:
uncommented them